Runtime/TraceAndOptimize.cs

```diff
@@ -27,6 +27,11 @@ internal class TraceAndOptimize : AvatarGlobalComponent
         [ToggleLeft]
         public bool preserveEndBone;
 
+        [NotKeyable]
+        [CL4EELocalized("TraceAndOptimize:prop:removeZeroSizedPolygons")]
+        [ToggleLeft]
+        public bool removeZeroSizedPolygons = true;
```

Comment:
```suggestion
        public bool removeZeroSizedPolygons = false;
```

Disabled by defaultにする?

Reply:
にしようか。副作用まぁ怖いし